setup.cfg

```diff
@@ -1,6 +1,6 @@
 [flake8]
 max-line-length = 110
-ignore = E133, E226, E228, N802, N803, N806
+ignore = E133, E226, E228, N802, N803, N806, N812, N813, N815, N816, W504
```

Comment:
Out of curiosity, why do we need the extra exceptions?

Reply:
Because I updated pep8-naming in February. I fixed all the lsst_distrib packages but I didn't fix ones I didn't know about.

Reply:
Ah.  OK.  Thanks for the explanation.